lib/admin/WP_Auth0_Admin_Features.php

```diff
@@ -55,131 +59,102 @@ public function init() {
     $this->init_option_section( '', 'features', $options );
   }
 
-  public function render_password_policy() {
-    $v = $this->options->get( 'password_policy' );
-?>
-      <input type="radio" name="<?php echo $this->options->get_options_name(); ?>[password_policy]" id="wpa0_password_policy_none" value="" <?php echo checked( $v, null, false ); ?>/><label for="wpa0_password_policy_none">None</label>
-      <input type="radio" name="<?php echo $this->options->get_options_name(); ?>[password_policy]" id="wpa0_password_policy_low" value="low" <?php echo checked( $v, 'low', false ); ?>/><label for="wpa0_password_policy_low">Low</label>
-      <input type="radio" name="<?php echo $this->options->get_options_name(); ?>[password_policy]" id="wpa0_password_policy_fair" value="fair" <?php echo checked( $v, 'fair', false ); ?>/><label for="wpa0_password_policy_fair">Fair</label>
-      <input type="radio" name="<?php echo $this->options->get_options_name(); ?>[password_policy]" id="wpa0_password_policy_good" value="good" <?php echo checked( $v, 'good', false ); ?>/><label for="wpa0_password_policy_good">Good</label>
-      <input type="radio" name="<?php echo $this->options->get_options_name(); ?>[password_policy]" id="wpa0_password_policy_excellent" value="excellent" <?php echo checked( $v, 'excellent', false ); ?>/><label for="wpa0_password_policy_excellent">Excellent</label>
-      <div class="subelement">
-        <span class="description">
-          <?php echo __( 'The difficulty of the user password where \'none\' requires a single character, \'low\' requires six characters and so on. For more details see our', 'wp-auth0' ); ?> <a target="_blank" href="https://auth0.com/docs/password-strength"><?php echo __( 'help page', 'wp-auth0' ); ?></a> <?php echo __( 'on password difficulty.', 'wp-auth0' ); ?>
-        </span>
-      </div>
-    <?php
+  public function render_password_policy( $args = array() ) {
+    $curr_val = $this->options->get( $args['opt_name'] );
+    $this->render_radio_button( $args['label_for'] . '_none', $args['opt_name'], '', 'None', empty( $curr_val ) );
```

Comment:
I'm not sure about this. But check if once the policy is enabled, if turning it OFF and then ON again still removes the policy object from the JSON or if it now does send a "none" value.

Reply:
I tested it back and forth between `none` and other values and everything works 👍